### PR TITLE
[Linux] Better paths handling

### PR DESCRIPTION
**Pull request by prophetl33t**

### CODE REVIEW

**Comment by ZingBallyhoo** [comment]
Okayge.

Code style is namesLikeThis
and spaces after commas

image

**Comment by prophetl33t** [comment]
Done :)

File: CASCEncDump/Program.cs [hunk 1]
```diff
@@ -19,14 +19,15 @@ namespace CASCEncDump {
...
 
-        private static string RawIdxDir => $"dump\\{BuildVersion}\\idx\\raw";
-        private static string RawEncDir => $"dump\\{BuildVersion}\\enc\\raw";
-        private static string ConvertIdxDir => $"dump\\{BuildVersion}\\idx\\convert";
-        private static string ConvertEncDir => $"dump\\{BuildVersion}\\enc\\convert";
-        private static string NonBLTEDir => $"dump\\{BuildVersion}\\nonblte";
-        private static string KeyFilesDir => $"dump\\{BuildVersion}\\keyfiles";
-        private static string AllCMFDir => $"dump\\{BuildVersion}\\allcmf";
-        private static string GUIDDir => $"dump\\{BuildVersion}\\guids";
+        private static string BaseDir => Path.Combine(Environment.CurrentDirectory, "dump", BuildVersion.ToString());
```
Comment (ZingBallyhoo) [inline CASCEncDump/Program.cs:22]: fixed

File: CASCEncDump/Program.cs [hunk 2]
```diff
@@ -19,14 +19,15 @@ namespace CASCEncDump {
...
+        private static string RawIdxDir => Path.Combine(BaseDir,"idx","raw");
```
Comment (ZingBallyhoo) [inline CASCEncDump/Program.cs:23]: notfixed

File: CASCEncDump/Program.cs [hunk 3]
```diff
@@ -19,14 +19,15 @@ namespace CASCEncDump {
...
+        private static string RawEncDir => Path.Combine(BaseDir,"enc","raw");
+        private static string ConvertIdxDir => Path.Combine(BaseDir,"idx","convert");
```
Comment (ZingBallyhoo) [inline CASCEncDump/Program.cs:25]: etc etc

**Review by ZingBallyhoo — CHANGES_REQUESTED** [review]

**Pull request merged**